InternTrack.Core.Api.Infrastructure.Provision/Brokers/Clouds/CloudBroker.ApplicationInsight.cs

```diff
@@ -0,0 +1,39 @@
+﻿// -------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE FOR THE WORLD
+// -------------------------------------------------------
+
+using Azure.Core;
+using Azure.ResourceManager.ApplicationInsights.Models;
+using Azure.ResourceManager.ApplicationInsights;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager;
+using Azure;
+
+namespace InternTrack.Core.Api.Infrastructure.Provision.Brokers.Clouds
+{
+    public partial class CloudBroker
+    {
+        public async ValueTask<ApplicationInsightsComponentResource> CreateApplicationInsightComponentAsync(
+            string componentName,
+            ResourceGroupResource resourceGroup)
+        {
+            ApplicationInsightsComponentCollection appInsightCollection = resourceGroup.GetApplicationInsightsComponents();
```

Comment:
```suggestion
            ApplicationInsightsComponentCollection appInsightCollection = resourceGroup.GetApplicationInsightsComponents();
```
            ApplicationInsightsComponentCollection appInsightCollection =
```             resourceGroup.GetApplicationInsightsComponents();

Reply:
I caught that yesterday too. Could've of sworn it got pushed. But thank you!